.github/workflows/ci.yml

```diff
@@ -43,12 +48,11 @@ jobs:
         sudo ./.ci/install_debian.sh
 
     - name: Dependencies [MacOS]
-      if: matrix.os == 'macOS-latest'
+      if: matrix.os == 'macOS-10.14'
       run: |
+        brew cask install xquartz
         brew install ace asio eigen tinyxml swig qt5 gsl pkg-config jpeg tinyxml
```

Comment:
'tinyxml' appears twice (actually also in the superbuild README).
What about: boost sqlite libedit ipopt opencv@3 ?